components/ArticlesList.js

```diff
@@ -56,7 +62,15 @@ const ArticlesList = ({ navigation }) => {
         <FlatList
           data={articles}
           renderItem={({ item }) => (
-            <View style={styles.card}>
+            <TouchableOpacity
+              style={styles.card}
+              onPress={() => {
+                navigation.navigate("article", {
+                  id: item.id,
+                  title: item.title,
```

Comment:
You don't need to pass the article title, since in the SingleArticle.js you use only the id to make the axios call
 `const fetchArticle = async () => {
    const { data } = await axios.get(
      "https://yesterdays-news-api.herokuapp.com/api/articles/${route.params.id}"
    );
    setArticle(data.article);
  };`